scripts/download.js

```diff
@@ -70,7 +70,11 @@ function download(version, fileName) {
   return new Promise((resolve, reject) => {
     get(url, (response) => {
       if (response.statusCode !== 200) {
-        throw new Error(`Not Found: ${url}`);
+        const errorDescription = [response.statusCode, response.statusMessage]
+          .filter((value) => value)
```

Comment:
Is this meant to filter out values like `null` or `undefined`?
If so,
You can simply use
```suggestion
          .filter(Boolean)
```

Reply:
Yep! Nice suggestion. I didn't know you could do that.